### DISCUSSION TITLE
Meta_Hours_9

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#2953] [Meta-Hours]
The sauce is coming soon, feel free to drop your sauce down 

![1_zMSR6lwGpWySHyxC5kagFA](https://user-images.githubusercontent.com/5887929/166962512-54a23fb7-cda6-40b0-944d-10c6a3e93c2a.jpeg)

### Post-covid recession
Markets. With recent market reset we would need adjust ✈️ plan, but honestly not too much as it's nothing we've anticipated is coming. We tried to have a small burn rate compared to others and save a bit on development to help us make the runway. Check burn_rate.md

Despite market downturns we would be doing another fundraising in a few weeks to keep us going and we still have a fair amount of runway for the next 18 months approx, but can still change despite how we go with the spending budget.

To make sure we can thrive and survive we will start looking on pre-revenue streams sooner than anticipated.

What's does means for us?
As any other portfolio company of investors we receive recommendations to hold on burn rate, where we in reality spend 1/10 of usual crypto start-up burn rate out there. Being highly efficient it's probably way we can afford not doing layoffs, not inherently freeze hiring and still looking for onboarding key roles as there will be plenty of good social capital on markers which would vibe 💓 with our 🧫 culture fit. So I would say timing is about right.

What's good for us?
I guess having a gig economy approach is way to go where we can still be capital and time efficient and rent talent whenever is needed and available with fair amount of core team helping us on internal Ops &c

### Dotsama Prague
I would like invite all who are listening that's we are organizing conference Dotsama conference in Prague where we would like to invite all, probably we will have some ticket raffles?
https://dotsama-prague.xyz

### Keep building
Hence our dev process is getting quite mature, I have no doubt that's right now would be our time to grow 🪴 our market share with scaling to other parachains. 

Currently in the works of very anticipated Basilisk, then there is Astar WASM and Acala EVM in works 😎

I guess once we will polish 💅✨ EVM implementation in approx 3-4 months with reserve inside Dotsama ecosystem I'm fairly confident we could run with our partners on any EVM based chain ⛓️ like Polygon or Ethereum if we really have that need.

Apart from scaling to outside, I still anticipating after Cambrian 💥 there will be consodilation of other marketplaces where their running costs are extremely high and revenue doesn't cover their expenses.

This is our opportunity to be patient and see where and when opportunity strikes.

To diversify, we will be implementing Reactive NFTs to bring among RMRK another protocol to open builders alternative, where with builders using Reactive NFTs have already open source UI available for their use and under license can fork it and use it for their usecase.

### 🍒 on top - SubWork
We've decided to go open co-work focused on Substrate builders in Bled, Slovenia. In beautiful mountain nature, next to the lake, away from everything, with pure focus and zero distraction.
We want to play a positive sum games and thought that's with more hive people at one place in person, more better thought exchange can happen and result more impressive things could be built easy as that.
At some point we would need our HQ anyway and we found that to lower expenditure, Lisbon is not place to be due surging demand there plus we are more athletic 👟 and for morning workout routines to chase wellbeing is place to be.

You're invited to come and show up!

https://subwork.xyz  

### Team efficacy

As I agree, team is our most valuable asset.
Everyone of you.
I'll try to spend my best time answering team members DMs and guiding through challenging stuff, offering advice and probably coaching as I want to dedicate 80% of my attention to spend quality time to grow 🪴 stuff together.

Getting into struggle? Feel free to reach out and I'll try do my best.
Being stucked on something? I'll help untwist for you.
😭 Emotional support? I'll try.

In background noise I've heard we might have team retreat (I don't count conference stuff) in July in Bled, Slovenia so will see how things goes! 
Would love to see you in person and importantly you can see each other in person.

### Ref 
https://twitter.com/EricNewcomer/status/1525541757177536512?t=Z6zNK0cQeGeS1BYyh821ZQ&s=09

**Comment by vikiival** [comment]
**Viki's Meta Minutes**
-

### DEV in numbers:
`is:pr is:merged created:>=2022-05-04`

> 39 PRs merged

### Basilisk:

**The BSX's branch contains more than 100 commits. 🎉**
Implementation status:

> UI:
- Create Collection ✅
- Mint NFT ✅ 
- Explorer ✅ 
- Profile (done when #2975 merged)
- Gallery detail ✅ 
- Collection detail

> Indexer:
- 1:1 with rubick ✅
- royalties ✅
- Offers

If you are keen to test the implementation. We have already a form for participation.

### Workshops:

- We had a workshop related to writing resolvers on the top of rubick. @KngZhi already made a couple of PRs to the rubick so h/t to him.
Also with the SubSquid team, we are planning to make a smaller workshop similar to the one we had last week.

There is also an upcoming workshop about Reactive NFTs at the Blockhaton. So don't forget to join us.

### Reading:
[We released an article which describes the implementation of rubick](https://medium.com/kodadot/index-your-kusama-nfts-b4fab5ae1628)

**Comment by damskyftw** [comment]
Damsky’s meta-hours will update you on:

- Coordination stuff
- Dotsama Prague
- Upcoming events
- Would like to give you a ticket :)
- Ambassadors
- NFT raffle
- Weather in Split

**Comment by DeluneClaire** [comment]
Clairee's mata_minutes :

- Dorahacks Amsterdot issue
- Basilisk form
- Twitter
- Gleam
- Instagram
- Kodaforest 
- Figma (Dotsama Prague design)

😶‍🌫️
